src/Components/ItemProperties.jsx

```diff
@@ -50,6 +50,22 @@ export default function ItemProperties({model, element}) {
 }
 
 
+/**
+ * If string is longer than maxWidth characters, wrap it in a tooltip.
+ * Otherwise wrap it in a paragraph.
+ * @param {string} str
+ * @param {Number} maxWidth (default 20)
+ * @return {Object} React component
+ */
+function paragraphMaybeWithTooltip(str, maxWidth=20) {
+  return (
+    str.length > maxWidth ?
+      <Tooltip title={str} placement='top'><p>{str}</p></Tooltip> :
+      <p>{str}</p>
```

Comment:
should it be replaced by 'Typography' to tie it to the theme?
the font is fine - maybe can be thinner?
<img width="768" alt="image" src="https://user-images.githubusercontent.com/3433606/158706966-501c1582-2786-42bb-8b33-9a8e434fec9f.png">